float64 relativeDirectionDegrees
float64 relativeSpeedKmph
